src/c++/perf_analyzer/test_custom_load_manager.cc

```diff
@@ -77,33 +77,29 @@ class TestCustomLoadManager : public TestLoadManagerBase,
     size_t intervals_index = 0;
 
     PauseWorkers();
-    InitCustomIntervals();
+    GenerateSchedule();
 
     // Keep calling GetNextTimestamp for the entire test_duration to make sure
     // the schedule is exactly as expected
     //
     while (expected_current_timestamp < max_test_duration) {
       for (auto worker : workers_) {
+        expected_current_timestamp += custom_intervals_[intervals_index];
```

Comment:
Why move this up?

Reply:
old schedule always started with a 0. That caused issues when wrapping, and doesn't really help much. For consistently now the first value is the first non-zero value of the schedule.